lib/exception_hunter/middleware/request_hunter.rb

```diff
@@ -0,0 +1,70 @@
+module ExceptionHunter
+  module Middleware
+    class RequestHunter
```

Comment:
why this change?

Reply:
Small refactor, we'll have many more middlewares in the future I thought it'd be nice to have a place for them